doc-Managing_Authentication/topics/External_auth.adoc

```diff
@@ -1,25 +1,26 @@
 [[external_auth]]
-= Configuring Federated Identity Management (External Authentication) with {product-title_short}
+= Configuring Identity Management (External Authentication) with {product-title_short}
 
-//Overview of federated auth in CloudForms here - benefits
 
-You can configure {product-title_short} to use federation authentication with system authentication methods such as Red Hat Identity Management (IdM), Red Hat Single Sign-On (SSO), or Active Directory (AD). 
+You can configure {product-title_short} to use federation authentication with system authentication methods such as Red Hat Identity Management (IdM) or IPA, Red Hat Single Sign-On (SSO), or Active Directory (AD). 
```

Comment:
Federation authentication - reads a bit awkwardly to me. Should this read "..to use federated authentication.."?

Reply:
Hmm... I looked up what the term actually means to figure out the right way to use the word and I might have used it incorrectly here. I've decided to remove any mentions of federated authentication to be safe.